AccessCheckoutSDK/AccessCheckoutSDK.xcodeproj/project.pbxproj

```diff
@@ -709,7 +709,7 @@
 				CODE_SIGN_IDENTITY = "";
 				CODE_SIGN_STYLE = Automatic;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = LPBQ5BR8NU;
+				DEVELOPMENT_TEAM = "";
```

Comment:
This seems related to Code Signing and I suspect that it may be safest to leave it as it is - https://groups.google.com/forum/#!topic/gyp-developer/E7UZYBjPNvQ